orca-webhook/src/test/groovy/com/netflix/spinnaker/orca/webhook/tasks/CreateWebhookTaskSpec.groovy

```diff
@@ -280,6 +280,39 @@ class CreateWebhookTaskSpec extends Specification {
     stage.context.statusEndpoint == null
   }
 
+  // Tests https://github.com/spinnaker/spinnaker/issues/2163
+  def "should evaluate statusUrlJsonPath for differing payloads"() {
+    setup:
+    def stage = new Stage(pipeline, "webhook", "My webhook", [
```

Comment:
In case you haven't see it, this could also be written as:

```
def stage = stage {
  context = [
    url: "https://...
    ...
  ]

}
```

The `stage {` comes from `ExecutionBuilder`